### ISSUE TITLE
AVT 1 - Text Area has DAP violation when disabled

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4540]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component and select the disable knob
Run DAP and the following violations are present (see screenshot):
<img width="947" alt="Screen Shot 2019-11-01 at 4 30 52 PM" src="https://user-images.githubusercontent.com/21676914/68057786-01c5ae00-fcc5-11e9-84f0-570d3fe668dc.png">

**Comment by abbeyhrt** [comment]
@dakahn do we have any guidance around contrast requirements for disabled states? I remember this coming up before but I don't remember what was landed on.

**Comment by snidersd** [comment]
@abbeyhrt @dakahn DAP is showing a contrast violation because the disabled state is not set.

**Comment by abbeyhrt** [comment]
Oh great! Thank you for clarifying! @snidersd

**Comment by dakahn** [comment]
DAP seems to be wrongly picking up the label text for this disabled input. The label is associated with the input in code via a `for` attribute -- unless i'm mistaken the label text for a disabled component doesn't fall within contrast requirements. Let me know if I'm wrong here @snidersd and I'll reopen. 

What I'm referencing
https://www.w3.org/TR/WCAG21/#contrast-minimum
https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by dakahn** [comment]
According to 1.4.3 the text should meet color contrast and references all "text in page".

**Comment by abbeyhrt** [comment]
@dakahn wait are you saying that the disabled text now must meet color contrast or that it needs to be marked as `aria-disabled` when disabled?

**Comment by dakahn** [comment]
After staring at 1.4.3 for a little longer it does in fact seem like this text is excluded from the contrast requirements. Closing it up 👍 thanks all

**Comment by snidersd** [comment]
@dakahn I've reported this issue to the [DAP Tools Repo](https://github.ibm.com/IBMa/Tools-Rules-HTML/issues/1271). issue # 1271
